File: Android/iOrder/app/src/main/java/hr/fer/oobl/iorder/iorder/injection/application/ApplicationComponentExposes.java
package hr.fer.oobl.iorder.iorder.injection.application;

import hr.fer.oobl.iorder.iorder.injection.application.modules.ApplicationModule;
import hr.fer.oobl.iorder.iorder.injection.application.modules.DataModule;
import hr.fer.oobl.iorder.iorder.injection.application.modules.DomainModule;
import hr.fer.oobl.iorder.iorder.injection.application.modules.MappersModule;
import hr.fer.oobl.iorder.iorder.injection.application.modules.ThreadingModule;
import hr.fer.oobl.iorder.iorder.injection.application.modules.UseCaseModule;

public interface ApplicationComponentExposes extends ApplicationModule.Exposes,
        DataModule.Exposes,
        DomainModule.Exposes,
        MappersModule.Exposes,
        ThreadingModule.Exposes,
        UseCaseModule.Exposes {
}

File: Android/iOrder/app/src/main/java/hr/fer/oobl/iorder/iorder/injection/application/ApplicationComponentExposes.kt
package hr.fer.oobl.iorder.iorder.injection.application

import hr.fer.oobl.iorder.iorder.injection.application.modules.ApplicationModule
import hr.fer.oobl.iorder.iorder.injection.application.modules.ThreadingModule

interface ApplicationComponentExposes : ApplicationModule.Exposes,
        //                                                     DataModule.Exposes,
        //                                                     DomainModule.Exposes,
        //                                                     MappersModule.Exposes,
        ThreadingModule.Exposes//                                                     UseCaseModule.Exposes
